kubernetes-server-mock/src/main/java/io/fabric8/kubernetes/client/server/mock/KubernetesCrudDispatcher.java

```diff
@@ -188,23 +189,23 @@ public MockResponse handlePatch(RecordedRequest request) {
 
         MediaType mergeType;
         if (contentType == null) {
-          mergeType = OperationSupport.JSON_PATCH;
+          mergeType = OkHttpClientImpl.JSON_PATCH;
```

Comment:
Why are we referencing the Okhttp implementation here?

Reply:
For now yes.  The mock server logic is still dependent on okhttp, that will introduce at least a test dependency.  This particular reference will be easy to remove/move as it's just a constant.